content/md/en/docs/tutorials/work-with-pallets/use-macros-in-a-custom-pallet.md

```diff
@@ -602,11 +416,19 @@ In this tutorial, you learned the basics of how to create a new custom pallet, i
 
 - How to compile and start a node that includes your custom pallet.
 
-- How you can add a React front-end component to expose the custom pallet to users.
+- How you can use the Polkadot JS Apps developer tool to interact with your custom blockchain.
 
 This tutorial covered the basics without diving too deeply into the code.
 However, there's much more you can do as you work toward building your own fully-customized blockchain.
 Custom pallets enable you to expose the features you want your blockchain to support.
 
+To complete your understanding of the proof-of-existence chain try:
+
+- Claiming the same file again with "ALICE" or even the "BOB" account.
+  - You should get an error!
+- Claiming other files with the "ALICE" and/or "BOB" accounts.
+- Revoking the claims with the appropriate claim owner account.
+- Looking at the final list of claims from reading storage.
+
```

Comment:
some follow up for the user. I would not mind if we did this explicitly with screenshots, but was too lazy here.